validator/README.md

```diff
@@ -75,15 +75,13 @@ In addition, install Node.js v4.4.2. E.g.,
 Dependencies:
 
 - npm
-- python 2.7 (should already be installed on OSX)
+- python 3 (should already be installed on OSX)
```

Comment:
On Travis, we successfully use python 3 on Linux. (See line 333 [here](https://travis-ci.org/github/ampproject/amphtml/jobs/733729029#L333).) Would it make sense to migrate this entire guide to python 3 (line 64)?

Reply:
Also, some answers to your questions in the PR description:

> Does python 3 come preinstalled with OSX?

The default version on my 2017 Macbook is python 2.7. This may be different for newer devices, since python 2.7 [was sunsetted](https://www.python.org/doc/sunset-python-2/) in Jan 2020.

> Does pip3 come preinstalled with python 3?

Based on [this doc](https://pip.pypa.io/en/stable/installing/#do-i-need-to-install-pip), the answer is yes.

Reply:
Yes, I think just moving to python3 makes sense. 2.7 has been officially end-of-life'd since Jan 1 2020. The script already supports it I believe.

Reply:
Updated to py3 in Linux, and updated py 3 install instructions for OSX. PTAL.